### PR TITLE
[AS7726-32X] Add to support mulit PSU SN in PDDF

### PR DESCRIPTION
**Pull request by jostar-yang**
Signed-off-by: Jostar Yang <jostar_yang@accton.com.tw>

#### Why I did it
This DUT support different PSU
#### How I did it
Add drv code to read from PSU eeprom to get model name and then get serial number
#### How to verify it
Test via pddf_psu cmd,
root@sonic:/home/admin# pddf_psuutil mfrinfo
PSU    Status    Manufacturer ID    Model       Serial              Fan Airflow Direction
-----  --------  -----------------  ----------  ------------------  -----------------------
PSU1   OK        ACBEL              FSF019-611  FSF0192105002864    EXHAUST
PSU2   OK        3Y POWER           YM-2651Y    SA070V581905000055  EXHAUST
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [x] 202106

### CODE REVIEW

**Review by FuzailBrcm — APPROVED** [review]

**Review by lguohan — APPROVED** [review]

File: platform/broadcom/sonic-platform-modules-accton/as7726-32x/modules/pddf_custom_psu.c [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+#include <linux/i2c.h>
+#include <linux/hwmon.h>
+#include <linux/hwmon-sysfs.h>
+#include <linux/err.h>
+#include <linux/delay.h>
+#include <linux/mutex.h>
+#include <linux/sysfs.h>
+#include <linux/slab.h>
+#include <linux/dmi.h>
+#include "../../../../pddf/i2c/modules/include/pddf_psu_defs.h"
```
Comment (lguohan) [inline platform/broadcom/sonic-platform-modules-accton/as7726-32x/modules/pddf_custom_psu.c:12]: there are too many pddf directory in the sonic-buildimage, we should add the include path in the search path and build.

**Review by lguohan — CHANGES_REQUESTED** [review]
:clock1:

File: platform/broadcom/sonic-platform-modules-accton/as7726-32x/modules/pddf_custom_psu.c (hunk 1 above)
Comment (FuzailBrcm) [reply platform/broadcom/sonic-platform-modules-accton/as7726-32x/modules/pddf_custom_psu.c:12]: @jostar-yang 
Can you take care of this comment?

File: platform/broadcom/sonic-platform-modules-accton/as7726-32x/modules/pddf_custom_psu.c [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+struct model_name_info models[] = {
+{PSU_TYPE_AC_110V, 0x20, 8, 8,  "YM-2651Y"},
+{PSU_TYPE_DC_48V,  0x20, 8, 8,  "YM-2651V"},
+{PSU_TYPE_DC_12V,  0x00, 11, 11, "PSU-12V-750"},
+{PSU_TYPE_AC_ACBEL_FSF019, 0x15, 10, 7, "FSF019-"}
+
+};
+
+struct serial_number_info serials[] = {
+{PSU_TYPE_AC_110V, 0x2e, 18, 18,  "YM-2651Y"},
```
Comment (FuzailBrcm) [inline platform/broadcom/sonic-platform-modules-accton/as7726-32x/modules/pddf_custom_psu.c:54]: Can you re-check this and correct the offset if needed?

Comment (jostar-yang) [reply platform/broadcom/sonic-platform-modules-accton/as7726-32x/modules/pddf_custom_psu.c:54]: as7726 support YM-2651Y PSU. So need this code.

File: device/accton/x86_64-accton_as7726_32x-r0/pddf/pddf-device.json [hunk 3]
```diff
@@ -1644,19 +1649,31 @@
...
 				{ "attr_name":"psu_p_out", "attr_devaddr":"0x58", "attr_devtype":"pmbus", "attr_offset":"0x96", "attr_mask":"0x0", "attr_cmpval":"0xff", "attr_len":"2"},
-				{ "attr_name":"psu_fan1_speed_rpm", "attr_devaddr":"0x58", "attr_devtype":"pmbus", "attr_offset":"0x90", "attr_mask":"0x0", "attr_cmpval":"0xff", "attr_len":"2"}
+				{ "attr_name":"psu_fan1_speed_rpm", "attr_devaddr":"0x58", "attr_devtype":"pmbus", "attr_offset":"0x90", "attr_mask":"0x0", "attr_cmpval":"0xff", "attr_len":"2"},
+				{ "attr_name":"psu_temp1_input", "attr_devaddr":"0x58", "attr_devtype":"pmbus", "attr_offset":"0x8d", "attr_mask":"0x0", "attr_cmpval":"0xff", "attr_len":"2"}
 			]
 		}
 	},
+	"PSU2-EEPROM":
+    {
+        "dev_info": { "device_type":"PSU-EEPROM", "device_name":"PSU1-EEPROM", "device_parent":"MUX7", "virt_parent":"PSU2"},
```
Comment (FuzailBrcm) [inline device/accton/x86_64-accton_as7726_32x-r0/pddf/pddf-device.json:1666]: "device_name":"PSU1-EEPROM" ---------> this should be "PSU2-EEPROM"

**Review by FuzailBrcm — APPROVED** [review]

**Review by prgeor — APPROVED** [review]

**Pull request merged**